nixpkgs-overlays/chromedriver/default.nix

```diff
@@ -0,0 +1,56 @@
+self: _: {
+    chromedriver = with self;
```

Comment:
Maybe not a blocker, but it might be nice to just make a PR to Nixpkgs' release-19.09 with this as well.

Reply:
@matthewbauer Yes we agreed about that. @cardenaso11 can make a Nixpkgs PR, put reflex-platform on it temporarily, and we can take a look.